src/bp/nlu-core/intents/svm-intent-classifier.ts

```diff
@@ -42,7 +42,7 @@ export class SvmIntentClassifier implements IntentClassifier {
       .filter(x => x.coordinates.filter(isNaN).length === 0)
       .value()
 
-    const classCount = _.uniq(points.map(p => p.label)).length
+    const classCount = _.uniqBy(points, p => p.label).length
```

Comment:
if you want to use uniqBy then `_.uniqBy(points, 'label')` is doing the trick. lodash doesn't enforce functions as accessors, when you pass in a string it uses its `_.get` func.

Reply:
Yeah I know but I really don't like this as its not type safe. Using an arrow function is uglier, but it will break if the property no longer exists.

I had problems in the past when using property name as strings instead of accessing them